### PR TITLE
feat: remove local x local y parser, move it to map_loader

### PR DESCRIPTION
**Pull request by tkhmy**
## Description
- parser for local_x and local_y will be move to map_loader instead of loading in lanelet
https://github.com/autowarefoundation/autoware/issues/3427
https://github.com/autowarefoundation/autoware.universe/pull/3200

## Tests performed

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by mitsudome-r** [comment]
Could you also modify documentation regarding local coordinate expression?
https://github.com/autowarefoundation/autoware_common/blob/main/tmp/lanelet2_extension/docs/lanelet2_format_extension.md#local-coordinate-expression

> Autoware Osm Parser will overwrite x,y positions with these tags when they are present.

Could you change it so that user would know that they should modify option in map_loader if they want to use local coordinates/

**Comment by tkhmy** [comment]
> Could you also modify documentation regarding local coordinate expression? https://github.com/autowarefoundation/autoware_common/blob/main/tmp/lanelet2_extension/docs/lanelet2_format_extension.md#local-coordinate-expression
> 
> > Autoware Osm Parser will overwrite x,y positions with these tags when they are present.
> 
> Could you change it so that user would know that they should modify option in map_loader if they want to use local coordinates?

I changed the docs in [56b6219](https://github.com/autowarefoundation/autoware_common/pull/174/commits/56b6219e6a7f9ef56a4cde92127120bd30b62b96)

File: tmp/lanelet2_extension/docs/lanelet2_format_extension.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Here is an example of MetaInfo in osm file:
 
 Sometimes users might want to create Lanelet2 maps that are not georeferenced.
 In such a case, users may use "local_x", "local_y" taggings to express local positions instead of latitude and longitude.
-Autoware Osm Parser will overwrite x,y positions with these tags when they are present.
+You will need to `lanelet2_map_projector_type` to `local`, then autoware map loader will overwrite x,y positions with these tags when they are present.
```
Comment (mitsudome-r) [inline tmp/lanelet2_extension/docs/lanelet2_format_extension.md:121]: Maybe putting a GitHub link to map_loader package might be user friendly.

**Review by isamu-takagi — APPROVED** [review]
LGTM

**Comment by tkhmy** [comment]
@mitsudome-r Can you please review it?

**Review by mitsudome-r — APPROVED** [review]

**Comment by tkhmy** [comment]
@mitsudome-r @isamu-takagi 
I do not have write access to this repository, can you give me the access or merge this PR?
Thanks

**Pull request merged**